internal/storage/image_test.go

```diff
@@ -157,16 +159,43 @@ var _ = t.Describe("Image", func() {
 			)
 
 			// When
-			names, err := sut.ResolveNames(ctx, testImageName)
+			names, err := sut.ResolveNames(
+				&types.SystemContext{
+					SystemRegistriesConfPath: "../../test/registries.conf",
+				},
+				testImageName,
+			)
 
 			// Then
 			Expect(err).To(BeNil())
 			Expect(names).To(Equal([]string{
-				testDockerRegistry + "/library/" + testImageName,
```

Comment:
It’s rather useful to test the `/library/` special case; the three replacement registries are all supposed to behave the same, so that’s not as interesting.